sql/mysql/Positive-Technologies/MySqlLexer.g4

```diff
@@ -1320,7 +1320,7 @@ STRING_USER_NAME:                    (
                                      );
 IP_ADDRESS:                          (
                                        [0-9]+ '.' [0-9.]+
-                                       | [0-9A-F:]+ ':' [0-9A-F:]+
+                                       | ([0-9A-F]+)? ':' ([0-9A-F]+)? ':' [0-9A-F:]+
```

Comment:
Please simplify:

```suggestion
                                       | [0-9A-F]* ':' [0-9A-F]* ':' [0-9A-F:]+
```

Reply:
ops, thanks for checking and correction, applied!